app/src/main/java/net/programmierecke/radiodroid2/utils/RecyclerItemMoveAndSwipeHelper.java

```diff
@@ -0,0 +1,44 @@
+package net.programmierecke.radiodroid2.utils;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+public class RecyclerItemMoveAndSwipeHelper<ViewHolderType extends SwipeableViewHolder> extends RecyclerItemSwipeHelper {
+
+    public interface MoveAndSwipeCallback<ViewHolderType> extends SwipeCallback<ViewHolderType>  {
+        void onMoved(ViewHolderType viewHolder, int from, int to);
+        void onMoveEnded(ViewHolderType viewHolder);
+    }
+
+    private MoveAndSwipeCallback<ViewHolderType> moveAndSwipeListener;
+
+    public RecyclerItemMoveAndSwipeHelper(int dragDirs, int swipeDirs, MoveAndSwipeCallback moveAndSwipeListener) {
+        super(dragDirs, swipeDirs, moveAndSwipeListener);
+        this.moveAndSwipeListener = moveAndSwipeListener;
```

Comment:
Unchecked assigments

Reply:
How can I fix this?

Reply:
Add annotation `@SuppressWarnings("unchecked")` on the method itself.